src/plugins/unified_search/public/filter_bar/filter_editor/index.tsx

```diff
@@ -84,16 +85,33 @@ class FilterEditorUI extends Component<Props, State> {
   }
 
   public render() {
+    // TODO: Hook up "Add vs Edit" panel titles
+    const panelTitleAdd = i18n.translate('unifiedSearch.filter.filterEditor.addFilterPopupTitle', {
+      defaultMessage: 'Add filter',
+    });
+    const panelTitleEdit = i18n.translate(
+      'unifiedSearch.filter.filterEditor.editFilterPopupTitle',
+      {
+        defaultMessage: 'Edit filter',
+      }
+    );
+
+    // TODO: Hook up "Add vs Update" button labels
+    const addButtonLabel = i18n.translate('unifiedSearch.filter.filterEditor.addButtonLabel', {
+      defaultMessage: 'Add filter',
+    });
+    const updateButtonLabel = i18n.translate(
+      'unifiedSearch.filter.filterEditor.updateButtonLabel',
+      {
+        defaultMessage: 'Update filter',
+      }
+    );
```

Comment:
Same with the "save" button, let's align the button text to the current action.

Reply:
Done https://github.com/elastic/kibana/pull/129877/commits/874133016587e680ac461bfa1799663888e906ff